design/Parts_for_3D_printing/README.md

```diff
@@ -0,0 +1,36 @@
+# Parts for 3D printing
+The above folders include editable Autodesk Fusion 360 files (.f3d) and STL files that can be 3D printed.
+
+## Anemometer case
+This case is to protect the Trisonica Mini ultrasonic wind anemometer that will be attached to the quadcopter
+
+<img width="300" src="Anemometer_parts/Anemometer_case.png">
+
+## The following parts are designed for the F450 quadcopter frame
+
+## Controller case
```

Comment:
What about the controller case?

Reply:
Was hoping to sort PCB's first in case we needed to make changes to the size or include more holes for wires etc

Reply:
Good point, so let's merge this now and we can come back to it later if need be